sample-apps/segmentation_spleen_postproc/lib/scribbles.py

```diff
@@ -192,33 +203,33 @@ class SpleenBIFSegSimpleCRF(SpleenPostProc):
     def __init__(
         self,
         dimension=3,
-        description="A post processing step with BIFSeg + SimpleCRF's CRF for Spleen segmentation",
+        description="A post processing step with ISeg + SimpleCRF's CRF for Spleen segmentation",
     ):
         super().__init__(dimension, description)
 
-    def pre_transforms(self):
-        return [
-            LoadImaged(keys=["image", "logits", "label"]),
-            AddChanneld(keys=["image", "logits", "label"]),
-            # at the moment Simple CRF implementation is bottleneck taking a long time,
-            # therefore scaling non-isotropic with big spacing
-            Spacingd(keys=["image", "logits"], pixdim=[3.5, 3.5, 5.0]),
-            Spacingd(keys=["label"], pixdim=[3.5, 3.5, 5.0], mode="nearest"),
-            ScaleIntensityRanged(keys="image", a_min=-300, a_max=200, b_min=0.0, b_max=1.0, clip=True),
-        ]
+    # def pre_transforms(self):
```

Comment:
any big chunk of unused code.. better to remove it

Reply:
Sure, I will remove this and any other similar chunks.